x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/rule_status.tsx

```diff
@@ -36,8 +37,15 @@ const RuleStatusComponent: React.FC<Props> = ({
   fieldName,
   isDraggable,
   value,
+  onClick,
 }) => {
   const color = useMemo(() => getOr('default', `${value}`, mapping), [value]);
+  const badge = (
```

Comment:
💅 💅 💅

Reply:
probably want to move this definition outside of this component

Reply:
You mean outside as a standalone component instead of inlining the component?